src/test/app/AccountTxPaging_test.cpp

```diff
@@ -271,6 +271,6 @@ class AccountTxPaging_test : public beast::unit_test::suite
     }
 };
 
-BEAST_DEFINE_TESTSUITE(AccountTxPaging, app, ripple);
+BEAST_DEFINE_TESTSUITE(AccountTxPaging, rpc, ripple);
```

Comment:
Is this change intentional?

Reply:
Yes, because it made more sense to me to have it under `rpc` and I figured I'd update it while I was here. If it's a breaking change or anything or just ill advised, I'm happy to change it back.

Reply:
Only problem is that we've already got a pending PR (#4794) that updates a bunch of unit tests to make them match their directory names. This would conflict with that, at least in principal, by moving away from that type of naming convention. If you think this makes more sense to be named `rpc`, it might make sense to move the files to `src/test/rpc`.

Reply:
Upon further review, I think I confused this test file with the `account_tx` RPC and this test file seems to be testing something else. I rolled back the change.